test/integration/rhel-8.0/data-read-mostly.patch.disabled

```diff
@@ -1,3 +1,5 @@
+Disabled due to https://github.com/dynup/kpatch/issues/940
```

Comment:
Perhaps a follow up commit could rework these tests to avoid functions that use jump labels?  I realize that's ignoring the problem, but would let us continue to test read-mostly and shadow variables in the meantime.

Reply:
I can have a look at it, but what might end up happening is that the test becomes completely different (patching other functions or maybe files) than the similar test in fedora/rhel folders. Which might be confusing if we keep the same name.

Perhaps we can keep this one disabled and add a second test "data-read-mostly-2.patch" that would (hopefully) work on RHEL-8 with the current state of kpatch, by avoiding to modify functions with jump labels.

Reply:
You're right, creating a newly named test file would be less-confusing.  Anyway, that was just an idea for a follow up PR if you wanted to explore that, not needed for this one.